### ISSUE TITLE
Substack feed works on iOS but not macOS

### ISSUE THREAD

**Issue by nehalvpatel** [Ranchero-Software/NetNewsWire#3296]
### Issue

I've been trying to get this feed to work on the macOS client: https://shesabeast.substack.com/feed/

I'm subscribed to it through **Feedly**. I have Feedly added to both the iOS and the macOS client.

Whenever a new article is published, it only shows up on the iOS version.

### macOS
<img width="310" alt="Screen Shot 2021-09-19 at 12 27 39 PM" src="https://user-images.githubusercontent.com/159021/133936950-6e3432ae-7302-43df-a933-73accde3f68d.png">

### iOS
<img width="310" src="https://user-images.githubusercontent.com/159021/133936978-930461fa-a521-4cd3-b173-10d5aaffc13d.jpeg">

**Comment by nehalvpatel** [comment]
Furthermore, when I paste in the feed URL in Safari and accept the option to open it in NetNewsWire, here's what happens:

<img width="516" alt="Screen Shot 2021-09-19 at 12 30 40 PM" src="https://user-images.githubusercontent.com/159021/133937055-e69ac7a4-34b2-47e4-a9b6-f30d8597b1ad.png">

<img width="528" alt="Screen Shot 2021-09-19 at 12 30 51 PM" src="https://user-images.githubusercontent.com/159021/133937064-5f15d3f5-73b3-4715-b746-a85c1c5466a7.png">

<img width="312" alt="Screen Shot 2021-09-19 at 12 30 34 PM" src="https://user-images.githubusercontent.com/159021/133937068-554cd1b3-8918-4fdb-9580-d1de9104f111.png">

**Comment by nehalvpatel** [comment]
Also, I was on version 6.0.2 (Build 609) on iOS and 6.0.3 (Build 6035) on macOS.

**Comment by vincode-io** [comment]
I'm not sure how to help you with this. It works fine for me.
<img width="1291" alt="Screen Shot 2021-09-25 at 3 27 08 PM" src="https://user-images.githubusercontent.com/16448027/134785204-b4a5687c-d8f5-4aaa-b8be-d7f14d2a8675.png">

It this feed not showing up in the sidebar at all for you? What happens if you search for "Oatmeal"? Does the latest article not show up in the finder?

**Comment by nehalvpatel** [comment]
Thanks for looking into this! Here is the search you requested:

<img width="1055" alt="Screen Shot 2021-09-25 at 5 16 37 PM" src="https://user-images.githubusercontent.com/159021/134787213-b2e4793e-a485-4f4c-be9b-ab19f5b2d484.png">

I added [another feed](http://matthewbischoff.com) to Feedly and had the same issue. I think it might be an issue that occurs when you start to have a large number of feeds on Feedly. (still odd that it only happens on the macOS client)

So, for both of our sanity I've resolved to just using iCloud for my feeds. I've moved all of them over and it's working fine now. (including the two feeds mentioned in this issue)

PS: I was bummed to see that the starred links do not move over when you move feeds to a different account. I ended up using some AppleScript to export the starred links into Raindrop.
